rockcraft.yaml

```diff
@@ -5,52 +5,66 @@ base: ubuntu:22.04 # the base environment for this ROCK
 version: "5.0.14-12" # just for humans. Semantic versioning is recommended
 summary: MongoDB in a rock. # 79 char long summary
 description: |
-    MongoDB is a source-available cross-platform
-    document-oriented database program. Classified
-    as a NoSQL database program, MongoDB uses JSON
-    -like documents with optional schemas.
+  MongoDB is a source-available cross-platform
+  document-oriented database program. Classified
+  as a NoSQL database program, MongoDB uses JSON
+  -like documents with optional schemas.
 license: Apache-2.0 # your application's SPDX license
 entrypoint:
```

Comment:
I could be wrong, but it looks like you still need to upgrade the ROCK to use Pebble as an entry point see [opensearch as an example](https://github.com/canonical/charmed-opensearch-rock/pull/14)
They are simple, but should take some time to test and verify

Reply:
Yes, we still do. It will be done in a separate PR